skrifa/src/outline/autohint/latin/blues.rs

```diff
@@ -436,6 +435,153 @@ fn compute_default_blues(font: &FontRef, coords: &[F2Dot14], style: &StyleClass)
     blues
 }
 
+/// Compute unscaled blue values for the CJK script set.
+///
+/// See <https://gitlab.freedesktop.org/freetype/freetype/-/blob/57617782464411201ce7bbc93b086c1b4d7d84a5/src/autofit/afcjk.c#L277>
+fn compute_cjk_blues(font: &FontRef, coords: &[F2Dot14], style: &StyleClass) -> [UnscaledBlues; 2] {
+    let mut blues = [UnscaledBlues::new(), UnscaledBlues::new()];
+    let mut outline_buf = UnscaledOutlineBuf::<MAX_INLINE_POINTS>::new();
+    let mut flats = [0; BLUE_STRING_MAX_LEN];
+    let mut fills = [0; BLUE_STRING_MAX_LEN];
```

Comment:
The blues functions are long and somewhat duplicative. Perhaps we can exploit. For example, the use of UnscaledOutlineBuf<256>, [i32; 51], [i32; 51] is common across both blues fns so I imagine we could do something like:

```rust
#[inline(always)]
fn buffers() -> (UnscaledOutlineBuf<MAX_INLINE_POINTS>, [i32; BLUE_STRING_MAX_LEN], [i32; BLUE_STRING_MAX_LEN]) {
    (UnscaledOutlineBuf::<MAX_INLINE_POINTS>::new(), [0; BLUE_STRING_MAX_LEN], [0; BLUE_STRING_MAX_LEN])
}
...
// compute_default_blues
let (mut outline_buf, mut flats, mut rounds) = buffers();
// compute_cjk_blues doesn't quite use the same names ... perhaps a comment explaining this should exist somewhere?
let (mut outline_buf, mut flats, mut fills) = buffers();


````